infrastructure/environments/cloudformation/full/la/s3.yaml

```diff
@@ -139,15 +139,19 @@ Resources:
             Action: "s3:GetObject"
             Resource:
               - !Join [ "", [ "arn:aws:s3:::", !Ref SharedBucket, "/*" ] ]
-          #- Effect: Allow
-          #  Principal:
-          #    AWS: !Ref OrgAccountRoleArn
-          #  Action:
-          #    - "s3:GetObject"
-          #    - "s3:ListBucket"
-          #  Resource:
-          #    - !Join [ "", [ "arn:aws:s3:::", !Ref SharedBucket, "/*" ] ]
-          #    - !Join [ "", [ "arn:aws:s3:::", !Ref SharedBucket ] ]
+          - Effect: Allow
+            Principal:
+              AWS: !Ref OrgAccountRoleArn
+            Action:
+              - "s3:GetObject"
+              - "s3:ListBucket"
+              - "s3:GetBucketAcl"
+              - "s3:GetObjectAcl"
+              - "s3:GetObjectAttributes"
+              - "s3:GetObjectTagging"
+            Resource:
+              - !Join [ "", [ "arn:aws:s3:::", !Ref SharedBucket, "/*" ] ]
+              - !Join [ "", [ "arn:aws:s3:::", !Ref SharedBucket ] ]
```

Comment:
Is it possible to do `SharedBucket.arn` the way you have for DataStoreBucket below?

Reply:
Yeah, that's cleaner for sure. Written at different times. Will switch to that one thanks.